File: 2021-06-22-spring-data-jpa-group-by/action-in-blog/src/main/java/blog/in/action/domain/ItemEntity.java
package blog.in.action.domain;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "TB_ITEM")
public class ItemEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private String name;
}

File: 2024-03-16-how-to-test-for-cacheable-test/action-in-blog/src/main/kotlin/blog/in/action/domain/ItemEntity.kt
package blog.`in`.action.domain

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
class ItemEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    val name: String = "",
    val categoryId: Int,
    val subCategoryId: Int
)